spec/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source_spec.rb

```diff
@@ -1,10 +1,177 @@
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource do
   let(:manager) do
-    FactoryBot.create(:provider_embedded_ansible, :with_authentication, :default_organization => 1).managers.first
+    FactoryBot.create(:provider_embedded_ansible, :default_organization => 1).managers.first
   end
+
+  # TODO:  Create a local repo instead... this will probably fail sporatically
+  # using a live repo
+  let(:params) do
+    {
+      :name    => "hello_world",
+      :scm_url => "https://github.com/NickLaMuro/ansible-tower-samples"
```

Comment:
Note:  I used my fork of this instead since I was able to add an "other_branch" for some other specs down below.

Reply:
For future self...original repo is here: https://github.com/ansible/ansible-tower-samples